spec.md

```diff
@@ -129,6 +129,19 @@ variant type.
 Every CloudEvent conforming to this specification MUST include one or more
 of the following context attributes.
 
+It could also include additional attributes in the "context attributes" that
```

Comment:
> CloudEvent producers MAY include additional extension attributes within the
event

Reply:
I will remove this duplication in the "Extension Attributes".  I think it is better to put into the generic "Context Attribute" so as not to restrict it to extensions

Reply:
@cathyhongzhang I think it might make more sense to put this into the other section (the extension) section since this is really about adding extensions.

Reply:
I am not sure if all future new context attributes belong to extensions. If some context attribute is used by a large number of use cases, then the WG might reach consensus to add it to the main spec.md directly. So I think putting this in the generic "context attributes" section, which includes both attributes endorsed by this WG (i.e. those explicitly defined in spec.md) and those that are not endorsed, is better.  What do you think?

Reply:
Currently none is defined in the spec.md, so by default, they all belong to extensions.

Reply:
I was just focused on the readability of it. This paragraph is really talking about extensions, granted its focused on "identity" extensions, but still extensions so it just felt a little out of place to have it here instead of in the extension section. Plus it then, sort of, reads like we're talking about extensions in 2 places instead of one.

In the end its not a big deal, it just felt a little odd, but I can live with it if others are ok with it.